lib/ui/profitloss/ProfitLossList.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:farmsmart_flutter/model/loading_status.dart';
+import 'package:farmsmart_flutter/redux/app/app_state.dart';
+import 'package:farmsmart_flutter/ui/profitloss/ProfitLossListItem.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:farmsmart_flutter/ui/common/headerAndFooterListView.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:redux/redux.dart';
+import 'mockRepositoryTryout/MockTransactionRepository.dart';
+import 'package:farmsmart_flutter/ui/profitloss/ProfitLossHeader.dart';
+
+class ProfitLossListViewModel {
+  LoadingStatus loadingStatus;
+  final String title;
+  final String detailText;
+
+  final List<ProfitLossListItemViewModel> transactions;
+
+  ProfitLossListViewModel({this.title, this.detailText, this.loadingStatus, this.transactions});
+
+  static ProfitLossListViewModel fromStore(Store<AppState> store) {
+    final mockViewModel = MockProfitLossListViewModel.build();
+
+    return mockViewModel;
```

Comment:
Join the return with the instance 
```suggestion
    return MockProfitLossListViewModel.build();;
```